### ISSUE TITLE
Account linking failed

### ISSUE THREAD

**Issue by smeevil** [y24o2/Alexa-Skill-for-Netatmo-Healthy-Home-Coach#3]
Hi, 

Thanks for the invite :)

After enabling the skill it asked to link my Netatmo account.
When trying to do this I get prompted by the following failure :

![78F64AFB-C8D5-4BD6-AC37-DB067CE24E17](https://user-images.githubusercontent.com/13874/61184131-5546ba00-a64a-11e9-9a55-c2bd25317360.png)

**Comment by y24o2** [comment]
Hello, is the scope 'read_homecoach' available for you netatmo account?

**Comment by joelez83** [comment]
Hi, congratulations on what you have done. I was trying to do a similar thing, but I can't complete the linking procedure (in Italy). What is the Api address? Is it available in Italy that you know of?

**Comment by y24o2** [comment]
you need to create a access_token and request
https://api.netatmo.com/api/gethomecoachsdata?access_token=access_token

i do it with a small function in [util.js](https://github.com/y24o2/Alexa-Skill-for-Netatmo-Healthy-Home-Coach/blob/master/Skill%20Code/lambda/util.js)

**Comment by joelez83** [comment]
Hey, thanks for the reply. What I don't understand (I'm not that good with this stuff), is the linking URI you should write on the alexa skill developer page. If I write api.netatmo.com, once I test the skill it results in error. Maybe I can't because I don't know how to create the access_token request, as you said... Here's my deal: The only think I want is to use my healthy home coach with alexa. Can you let me use your skill? I can translate all the part needed in italian, so you can implement it with italian language!

**Comment by y24o2** [comment]
i added these url to the skill config "Account Linking":

Your Web Authorization URI: https://api.netatmo.com/oauth2/authorize

Access Token URI: https://api.netatmo.com/oauth2/token

i'll can't connect you to my skill. the beta test is closed and time is over, but my skill is not public for some reasons.
if you know how, i'll let you use it (and add some italien phrases to the skill set)

**Comment by joelez83** [comment]
Thank you very much. I don't know why your skill is not available yet, and still don't know how to manage...but thank you for your help so far!

**Comment by joelez83** [comment]
I did it! Finally I could link my netatmo account to the skill, but there is another error. I asked Netatmo, because in the link I see my weather station and not my home coach. Let's see if it's something referred to them...

**Comment by y24o2** [comment]
I think you have to edit (or set) the scope in your netatmo oauth config.

**Comment by joelez83** [comment]
Yes! That was the problem and now it works. But now there's this 
<img width="375" alt="image" src="https://user-images.githubusercontent.com/73608054/97691539-c5c35580-1a9e-11eb-8d93-f51d2e1d380e.png">
Don't know what to do...
